components/application/widgets/DatePicker.js

```diff
@@ -99,7 +99,7 @@ function DatePicker({ onDateSelect, data }) {
                   >
                     <span
                       className={
-                        selected > 0 && selected === itemDays
+                        selected > 0 && selected == itemDays
```

Comment:
Why using `==` here rather than `===`?

Reply:
If use` ===`, the selected style will not take effect.